Classes/Event/EventMetadata.php

```diff
@@ -11,46 +11,25 @@
  * source code.
  */
 
-use Neos\Cqrs\Domain\Timestamp;
-use TYPO3\Flow\Annotations as Flow;
-
 /**
- * The MessageMetadata is a container for arbitrary metadata for Commands and Events.
+ * The EventMetadata is a container for arbitrary metadata for Events.
```

Comment:
> Not entirely sure about that

OK, my point was rather: That Commands and Events are both messages is a geeky technical detail ;)
We could also just call it `Metadata`, just not introduce yet another term.

> I'd however highly vote for making the event-specific base metadata (version, aggregateId, occuredAt) fixed part of the metadata

yes, I agree.. even though `aggregateId` is no longer part of the metadata as of now.
I suggest to add the constants for version & timestamp/occuredAt to the Metadata object itself, so that they are predefined by the framework.
Still they should be set via the same mechanism like any other metadata IMO (e.g. via signal/slots)